File: Assignment/Module-1(fundamentals/p9.cpp
#include<stdio.h>
main()
{
	int a,b;
	printf("Enter the number :");
	scanf("%d%d",&a,&b);
	printf("before swaping: a=%d,b=%d \n",a,b);
	a=a+b;
	b=a-b;
	a=a-b;
	printf("after swaping: a=%d,b=%d \n",a,b);
}

File: Assignment/flutter dart assignment module-1/p9.dart
import 'dart:io';

void main() {
  int num1, num2;

  stdout.write("Enter the first number: ");
  num1 = int.parse(stdin.readLineSync()!);

  stdout.write("Enter the second number: ");
  num2 = int.parse(stdin.readLineSync()!);

  num1 = num1 + num2;
  num2 = num1 - num2;
  num1 = num1 - num2;

  print("After swapping:");
  print("First number: $num1");
  print("Second number: $num2");
}
